torchgeo/datasets/oscd.py

```diff
@@ -311,3 +314,190 @@ def get_masked(img: Tensor) -> Array:  # type: ignore[type-arg]
             plt.suptitle(suptitle)
 
         return fig
+
+
+class OSCDDataModule(pl.LightningDataModule):
+    """LightningDataModule implementation for the OSCD dataset.
+
+    Uses the train/val/test splits from the dataset.
+
+    .. versionadded: 0.2
+    """
+
+    # NOTE: For some reason this doesn't have the B10 band so for now I'll insert
+    # a value based on it's neighbor while I figure out what to put there.
+
+    # (B01, B02, B03, B04, B05, B06, B07, B08, B8A, B09, B10, B11, B12)
+    # min/max band statistics computed on 100k random samples
```

Comment:
Why not just take the statistics for the entire train set?

Reply:
I think this is copy/paste from BigEarthNet

Reply:
Ah, this should probably be removed then.